README.md

```diff
@@ -0,0 +1,150 @@
+-->
+
+<!--  There are a few todo's so take care of each one -->
+
+<!--
+*** To upload screenshots, one way is:
+*** Create a branch named "static-assets" by running:
+***    > git checkout --orphan static-assets
+*** The --orphan flag creates a new branch but without any prior commits.
+*** However it does still keep the working tree and index.
+*** So delete those files by running (watchout for the dot at the end):
+***    > git rm -rf .
+*** Add and commit the static files:
+***    > git add screenshot.png demo.gif logo.png
+***    > git commit -m "Added Assets"
+*** Push the branch to remote:
+***    > git push origin static-assets
+*** Name all future commit as "Updated Assets"
+*** To display the picture, in the README, type:
+*** ![Alt Text](../static-assets/screenshot_name.png?raw=true)
+-->
+
+<!-- DON'T FORGET TO CHANGE THE YEAR AND THE NAME OF COPYRIGHT HOLDER IN THE LICENSE (LINE 3)-->
+
+<div align="center">
+  <h1>json-csv-converter</h1>
+  <p><i>CLI tool for converting JSON to CSV files.</i></p>
+</div>
+
+<div align="center">
+  <a href="https://github.com/furiousdom/json-csv-converter/stargazers">
+    <img src="https://img.shields.io/github/stars/elangosundar/json-csv-converter?style=for-the-badge" alt="Stars Badge" />
+  </a>
+  <a href="https://github.com/furiousdom/json-csv-converter/network/members">
+    <img src="https://img.shields.io/github/forks/furiousdom/json-csv-converter?style=for-the-badge" alt="Forks Badge" />
+  </a>
+  <a href="https://github.com/furiousdom/json-csv-converter/graphs/contributors">
+    <img src="https://img.shields.io/github/contributors/furiousdom/json-csv-converter?style=for-the-badge" alt="GitHub Contributors Badge" />
+  </a>
+  <a href="https://github.com/furiousdom/json-csv-converter/pulls">
+    <img src="https://img.shields.io/github/issues-pr/furiousdom/json-csv-converter?style=for-the-badge" alt="Pull Requests Badge" />
+  </a>
+  <a href="https://github.com/furiousdom/json-csv-converter/issues">
+    <img src="https://img.shields.io/github/issues/furiousdom/json-csv-converter?style=for-the-badge" alt="Issues Badge" />
+  </a>
+  <a href="https://github.com/furiousdom/json-csv-converter/blob/master/LICENSE">
+    <img src="https://img.shields.io/github/license/furiousdom/json-csv-converter?style=for-the-badge" alt="License Badge" />
+  </a>
+</div>
+
+<br />
+
+<div align="center">
+  <p>
+    <a href="https://github.com/furiousdom/json-csv-converter/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/furiousdom/json-csv-converter/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+## About The Project
+
+<!-- TODO: CHANGE THE EXAMPLE URL (LINKING TO DEMO) BELOW OR REMOVE IT COMPLETELY -->
+<!-- TODO: LINK TO ACTUAL SCREENSHOT OR DELETE THE WHOLE LINE -->
+[![json-csv-converter Screen Shot](../static-assets/screenshot_name.png)](https://example.com)
+
+<!-- TODO: DELETE LINE BELOW AND THIS COMMENT. -->
+<!-- TODO: WRITE DETAILED PROJECT DESCRIPTION TEXT HERE -->
```

Comment:
Is the plan to do this in this PR?

Reply:
No. This shouldn't have been left hanging. Mistake on my part which I have fixed in the next commit.